reqs/requirements-dev-lite.txt

```diff
@@ -3,6 +3,7 @@ coverage[toml]>=7.5.1
 flaky>=3.7.0 # for docs-integration tests that access cloud-resources that access cloud-resources
 flask>=1.0.0 # for s3 test only (with moto)
 freezegun>=0.3.15
+jsonschema>=4.23.0
```

Comment:
Do we need to bump our actual version in `requirements.txt`? Why are we bound to `2.5.1` there?

Reply:
Oh nice I didn't see that was there! I removed this change and left the dep alone.

Reply:
We don't need to bump it and I don't see what the benefit would be.